exp/api/v1beta2/awsmanagedmachinepool_types.go

```diff
@@ -199,6 +199,10 @@ type AWSManagedMachinePoolStatus struct {
 	// +optional
 	LaunchTemplateVersion *string `json:"launchTemplateVersion,omitempty"`
 
+	// InfrastructureMachineKind is the kind of the infrastructure resources behind MachinePool Machines.
+	// +optional
+	InfrastructureMachineKind string `json:"infrastructureMachineKind,omitempty"`
```

Comment:
I don't think we should have machines for `AWSManagedMachinePool` as the ASG created for an EKS managed node group is managed by AWS.

Reply:
I removed all EKS-related code

Reply:
Thanks @AndiDog , resolving this.